policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
```

Comment:
Is this list not yet filled out? Otherwise I'm not sure I understand what this is communicating.

Reply:
This is intended to be a set of the required sections that the role description should cover, not something that gets filled out inline.

Reply:
Can you make that explicit?

---
Review Role Description (https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944). A role description should contain the following content at a minimum:
- Aims
- Estimated Time Investment
- Responsibilities
- Qualifications
- Term
---
(As a side note, I find the term "aims" to be confusing. I've not seen it used before in this way and wondering if we can use goals instead much like the committee descriptions do?)